### PR TITLE
Parameter Sweep over a Representative Set of Latent Space Dimensionality

### PR DESCRIPTION
**Pull request by gwaygenomics**
This pull request is a precursor to the larger plan of sweeping over many more latent space combinations. The purpose of this pull request is to test hyperparameter stability while varying the latent space dimensionality in adage and tybalt models.

When we perform the larger sweep, we want to be sure that the results are not impacted by poor hyperparameter combinations.

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
I have no comments about the code @gwaygenomics, but let's see what @danich1 has to say about it. I am assuming that this smaller set hyperparameter choices (and how they differ for each of the models) is based on some prior experience. You might consider recording that here for posterity.

File: config/z_parameter_sweep_adage.tsv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+parameter	sweep
```
Comment (danich1) [inline config/z_parameter_sweep_adage.tsv]: Can we change sweep to values or something a lil more intuitive?

File: scripts/num_components_paramsweep.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+if not os.path.exists(param_folder):
+    os.makedirs(param_folder)
+
+# Load data
+parameter_df = pd.read_table(parameter_file, index_col=0)
+config_df = pd.read_table(config_file, index_col=0)
+
+# Retrieve hyperparameters to sweep over
+num_components = get_param('num_components')
```
Comment (danich1) [inline scripts/num_components_paramsweep.py]: Are you relying on panda's string to number conversion for this block? Thats fine, but may want to be explicit on how one creates the parameter file.

Comment (gwaygenomics) [reply scripts/num_components_paramsweep.py]: Sounds good. Right now I rely on scripts that ingest these parameters to perform the appropriate conversions. I bolstered the documentation in a recent commit.

File: config/z_parameter_sweep_adage.tsv (hunk 1 above)
Comment (gwaygenomics) [reply config/z_parameter_sweep_adage.tsv]: updated to `sweep_values`

**Comment by gwaygenomics** [comment]
Thanks for comments @danich1 - I have addressed your comments (and added some changes of my own) to the PR. Ready for review again!

**Pull request merged**